privacy/index.md

```diff
@@ -4,150 +4,562 @@ title: Privacy Policy
+Data is stored in a protected database on a specially guarded server.
+The server is operated by us and hosted by Hetzner.
 
-- Pages visited
-- IP address
-- Registration IP address (linked to the account until deletion)
-- Email address
-- Incoming and outgoing links
-- Reading times per Post
-- User profiles viewed
-- Likes given and flags being requested
+**Data Linking:**
 
-To give you the most privacy possible we allow you to look into most of the forum without being logged in.
+Information/data used for the purpose of linking payments to a specific
+user to enable services are as follows:
 
-*Note: We are looking into options to make Discourse more privacy friendly, if you ideas please get in touch*
+-   Valid email address, if or when required.
 
-In general we encourage everyone not to use their real name and not to use their primary/private email address.
+-   User name, not your real name.
 
-<!-- #### Portmaster* -->
-<!-- #### Gate17* -->
+-   Payment identifiers, transaction code sent to us via the payment
+    gateway used.
 
-### Information
+**Data Retention:**
 
-The following sections will detail what exactly we mean by the rather short descriptions of information used before.
+we are required by law to keep payment records for a period of 7 years,
+for taxation purposes, even in the event that the account is deleted, we
+are required to keep the transaction identifiers for this period.
 
-#### User Agent
+Payment records are only linked to your account until it is deleted, or
+expired, they then become orphaned and cannot be linked back to your
+account.
 
-We collect what operating system and web browser is used.
-The goal is to develop software that our customers can use. Therefore, knowing which operating systems they use is important for effective development.
+Payments of up to 400€ (incl. Taxes) do not require customer
+information, i.e. your invoices will be anonymous.
 
-#### Language
+**Data Protection:**
 
-We look at the language setting of your system (or browser).
-To achieve the best privacy possible, it is important that the customer understands what each setting of our product is doing. This means working with them in their language. Knowing what languages (future) customers use, helps us to prioritize translation efforts.
+In Transit: All traffic between the Customer and Safing ICS Technologies
+GmbH is encrypted using modern Transport Layer Security (TLS). TLS is a
+cryptographic protocol designed to provide communications security over
+a computer network and aims primarily to provide confidentiality and
+data integrity between two communicating computer applications.
 
-#### Location
+At rest: Hot data (in use) is protected on our server, cold data
+(backups) are encrypted.
 
-We use your IP address to identify the country from which you are accessing our services. Only half of your IP address saved.
+**Article 4. How we use your PII & Data**
+-----------------------------------------
```

Comment:
Is this valid? You are making it bold _and_ a header.
I would suggest always using `##` in the correct level, as they are easier to edit and read. Especially if `#`s are used elsewhere.